content/en/_index.html

```diff
@@ -29,7 +33,7 @@ <h3 class="section-head">
       <span class="border-bottom pb-2">What is Kubeflow?</span>
     </h3>
     <p class="mx-auto col-11 col-xl-7 px-0">
-      Kubeflow makes artificial intelligence and machine learning simple, portable, and scalable.
+      Kubeflow is a platform that makes artificial intelligence and machine learning simple, portable, and scalable.
```

Comment:
Same as my other comment, we are not a platform, but a collection of tools which can form a platform.

Also, I think the proposed text is less punchy from a marketing perspective.

We are trying to sell the "why" not the "what" in this sentence.

Reply:
>a collection of tools which can form a platform

To me, that's equivalent. Most people using Kubeflow consider it a platform and we should just lean into that.

Reply:
This sentence went through many iterations recently, and I am not sure we need to re-litigate it.

I am happy with the current state:

> Kubeflow makes artificial intelligence and machine learning simple, portable, and scalable. We are an ecosystem of [Kubernetes](https://kubernetes.io/) based components for each stage in [the AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle) with support for best-in-class open source [tools and frameworks](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem).